FTPropertyWrappers.podspec

```
@@ -1,10 +1,9 @@
 Pod::Spec.new do |s|
   s.name         = "FTPropertyWrappers"
-  s.version      = "1.1.0"
+  s.version      = "1.2.0"
```

The API introduced in this PR is incompatible with the old one. 

> MAJOR version when  you make incompatible API changes,
> MINOR version when you add functionality in a backwards compatible manner, and
> PATCH version when you make backwards compatible bug fixes.
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
https://semver.org

We should release this as a version 2.0.0.

---

I would also ask the team for ideas on more complex property wrappers that should be included in this repo. I'm not talking about things like `Clamping` but something more challenging (like the Keychain is).

I am on the fence about having 1 property wrapper in a repo called FRPropertyWrapper`S`.

---

Sure, let's release it as major version.

```suggestion
  s.version      = "2.0.0"
```

I agree that the name did not age well. But I am strongly against forcefully adding new property wrappers only to name makes sense. There are still two useful property wrappers remaining – I know they are both related to keychain, but I am not sure it's reasanable to rename the package.